discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -12,17 +12,20 @@
 from test_infra.consts import resources
 from test_infra.controllers.node_controllers.libvirt_controller import LibvirtController
 from test_infra.controllers.node_controllers.node import Node
-from test_infra.helper_classes.config import BaseTerraformConfig
+from test_infra.helper_classes.config import BaseTerraformConfig, BaseClusterConfig
 from test_infra.tools import static_network, terraform_utils
+from test_infra.utils.cluster_name import get_cluster_name_suffix
 
 
 class TerraformController(LibvirtController):
 
-    def __init__(self, config: BaseTerraformConfig):
+    def __init__(self, config: BaseTerraformConfig, cluster_config: BaseClusterConfig):
         super().__init__(config.private_ssh_key_path)
-        self.config = config
-        self.cluster_name = str(config.cluster_name)
-        self.network_name = config.network_name + config.cluster_name.suffix
+        self.config: BaseTerraformConfig = config
+        self._cluster_config: BaseClusterConfig = cluster_config
```

Comment:
same

Reply:
Those are 2 different annotations. One is for the `__init__` arguments and the other is for the class properties.
They are not  mandatory so we can remove them if you want :)